prow/jobs/jobs.go

```diff
@@ -306,7 +325,30 @@ func (ja *JobAgent) loadPostsubmits(path string) error {
 	return nil
 }
 
-func (ja *JobAgent) tryLoad(pre, post string) {
+// Hold the lock.
+func (ja *JobAgent) loadCrons(path string) error {
+	b, err := ioutil.ReadFile(path)
+	if err != nil {
+		return err
+	}
+	nj := map[string][]Cron{}
+	if err := yaml.Unmarshal(b, &nj); err != nil {
+		return err
+	}
+	for _, js := range nj {
+		for j := range js {
+			d, err := time.ParseDuration(js[j].Interval)
```

Comment:
No, that's why it's "interval". Rather than saying a cron string you say a golang interval, such as `5m` or `1h30m`.